### PR TITLE
Single item or full page

### PR DESCRIPTION
**Pull request by BenjaminCall**
# Code-review checklist #

  * Documentation
      * [ ] Interfaces describe how other programmers would use them.
      * [ ] Code that is not obvious has documentation describing why
        that code is the way it is.

### CODE REVIEW

File: client/js/_author/components/assessments/_edit_assessment.jsx [hunk 1]
```diff
@@ -68,7 +65,7 @@ export class EditAssessment extends React.Component {
   };
 
   componentDidMount() {
-    this.props.getAssessments(this.props.params.bankId);
+    // this.props.getAssessments(this.props.params.bankId);
```
Comment (BenHSmith13) [inline client/js/_author/components/assessments/_edit_assessment.jsx]: just remove

File: client/js/_author/components/assessments/assessment_form.jsx [hunk 2]
```diff
@@ -55,18 +57,27 @@ export default class AssessmentForm extends React.Component {
     this.props.updateItemOrder(itemIds);
   }
 
+  showSinglePageOption() {
+    if (this.props.publishedAndOffered) {
+      return (
+        <div className="o-item__top">
+          <div className="c-checkbox u-right">
+            <input type="checkbox" id="check01" name="check" onChange={e => this.props.updateSingleItemOrPage(e.target.checked)} />
```
Comment (BenHSmith13) [inline client/js/_author/components/assessments/assessment_form.jsx]: You will need a more unique id

File: client/js/_author/reducers/assessments.js [hunk 3]
```diff
@@ -48,13 +48,28 @@ export default function banks(state = initialState, action) {
...
+
     case 'DELETE_ASSESSMENT_DONE': {
       const newState = _.cloneDeep(state);
       const bankId = action.original.bankId;
       delete newState[bankId][action.original.assessmentId];
       return newState;
     }
 
+    case 'UPDATE_SINGLE_ITEM_OR_PAGE_DONE': {
+      debugger
```
Comment (BenHSmith13) [inline client/js/_author/reducers/assessments.js]: bugger, and empty?

File: client/js/actions/qbank/assessments.js [hunk 4]
```diff
@@ -84,6 +85,7 @@ export function createAssessmentOffered(bankId, assessmentId) {
   return {
     bankId,
     assessmentId,
+    body    : { id: '123' }, // have to send up garbage data to make it work.
```
Comment (BenHSmith13) [inline client/js/actions/qbank/assessments.js]: does this ID get reset or does it come back as the object id?
If we need to keep this id, I have a guid function in the utils

File: client/js/_author/components/assessments/assessment_form.jsx (hunk 2 above)
Comment (BenjaminCall) [inline client/js/_author/components/assessments/assessment_form.jsx]: why? there will only ever be one on the page

File: client/styles/_author/partials/_dashboard.scss [hunk 5]
```diff
@@ -172,3 +172,21 @@
...
+    // height: 3rem;
+    border-radius: $radius;
+    width: 100%;
+    padding: 0 1rem;
+
+    &:focus{
+      outline: thin dotted;
+    }
+  }
+}
```
Comment (BenHSmith13) [inline client/styles/_author/partials/_dashboard.scss]: newLine

File: client/js/actions/qbank/assessments.js (hunk 4 above)
Comment (BenjaminCall) [inline client/js/actions/qbank/assessments.js]: no nothing happens with this ID, it is garbage. Just needs any value sent up in the body with a value for it to work. I could send name: "billy". just needs something

File: client/styles/_author/partials/_dashboard.scss [hunk 6]
```diff
@@ -189,4 +189,8 @@
       outline: thin dotted;
     }
   }
-}
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (BenHSmith13) [inline client/styles/_author/partials/_dashboard.scss]: This?

**Comment by BenHSmith13** [comment]
# LGTM
lgtm

**Pull request merged**